src/søknad/steg/5-aktivitet/underUtdanning/StudieArbeidsmengde.tsx

```diff
@@ -29,6 +35,10 @@ const StudieArbeidsmengde: React.FC<Props> = ({
 
   const arbeidsmengdeLabel = hentTekst('utdanning.label.arbeidsmengde', intl);
 
+  const erIkkeUndefinedOgMerEnnNittiNiProsent =
```

Comment:
Må vi ikke sjekke at det er deltid?

Reply:
Eller vises ikke denne komponenten for heltid?

Reply:
erIkkeUndefinedOgMerEnnNittiNiProsent => er dette egentlig "erHeltid"